src/types/position.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from 'zod';
+
+import { AssetSchema } from './assets.js';
+import { LoanSchema } from './offers.js';
+import { PagingSchema } from './paging.js';
+
+export const LoanPositionSchema = LoanSchema.extend({
+  created: z.number(),
+  id: z.string(),
+});
+
+export const PositionSchema = z.object({
+  asset: AssetSchema,
+  loan: LoanPositionSchema,
+});
+export type Position = z.infer<typeof PositionSchema>;
+
+export const PositionsResponseSchema = z.object({
+  paging: PagingSchema,
+  positions: PositionSchema.array(),
+});
+export type PositionsResponse = z.infer<typeof PositionsResponseSchema>;
```

Comment:
The naming could be more understandable imo.

We should rename `LoanSchema` to `LoanBaseSchema` and use that for `AssetOffersSchema` as well as below:

```suggestion
export const LoanSchema = LoanBaseSchema.extend({
  created: z.number(),
  id: z.string(),
});

export const LoanPositionSchema = z.object({
  asset: AssetSchema,
  loan: LoanSchema,
});
export type LoanPosition = z.infer<typeof LoanPositionSchema>;

export const LoanPositionsResponseSchema = z.object({
  paging: PagingSchema,
  positions: LoanPositionSchema.array(),
});
export type LoanPositionsResponse = z.infer<typeof LoanPositionsResponseSchema>;
```